### DISCUSSION TITLE
Code formating preference for if statement

### DISCUSSION THREAD

**Discussion by Cfun1** [xamarin/XamarinCommunityToolkit#1148] [General]
I noticed sometimes we are using curly brackets in an if statement sometimes no, the current .editorconfig recommends using curly brackets brackets even for one single statement https://github.com/xamarin/XamarinCommunityToolkit/pull/884

Should we follow .editorconfig recommendation in all cases or there are some exceptions ?

**Comment by AndreiMisiukevich** [comment]
Well, since our code convention is changed, then yes, we should.)

**Reply by Cfun1** [reply]
What do you mean by "code convention is changed? I believe .editorconfig was from the beginning this way.

**Reply by AndreiMisiukevich** [reply]
I mean, that when we were beginning this project we reached an agreement that we should avoid brackets for one-line if-else statements.
